ddtrace/tracer/tracer.go

```diff
@@ -95,6 +96,15 @@ type tracer struct {
 
 	// statsd is used for tracking metrics associated with the runtime and the tracer.
 	statsd statsdClient
+
+	// abandonedSpans holds a linked list of potentially abandoned spans for all traces.
+	abandonedSpans *list.List
+
+	// cIn receives spans when they are created to be added to abandonedSpans
```

Comment:
Is this more the tracer `sending` spans as they are created not receiving them? (Although I guess the abandoned span checked is receiving them... 🤔 )

Reply:
I guess it's technically sending spans, but the channel is... receiving spans once they are created... but then its sending these received spans to the abandoned spans list... 😵 I'll just clarify these comments a bit LOL